src/test/actions.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-FileCopyrightText: © 2021 Dai Foundation <www.daifoundation.org>
```

Comment:
My main issue with this refactor is that it introduces yet another file which needs to be modified and reviewed, potentially in every single spell.

While I get the separation of concerns in principle, I believe it does not work well with the nature of spell crafting. We try to follow the principle of co-location to reduce the already high cognitive overhead involved in the task.

So I believe there is little to no gain making these changes, it's just moving code around.

Reply:
Indeed, the point being that instead of having to check in [2023-11-29](https://github.com/makerdao/spells-mainnet/pull/372/files#diff-cd597c47b9a6e2bae57aab767ee9e3d5bb3d8edeb3a2248b4519b9af791cb468) changes in `DssSpell.t.sol` scattered in lines L405-423, L475-477, L523-526, L546-555 & L590-603, a reviewer should have to check only 1 file, to make [contrast the data with the Exec Doc](https://github.com/makerdao/pe-checklists/blob/8e075342d60fac1621673768a76ad797107fbee7/spell/spell-crafter-mainnet-workflow.md?plain=1#L116).

On a further development, that file could be cleanup more easily, resulting on less changes on the test file, making each change raise more alerts, rather than being used the reviewer to see changes.